libobs/obs.h

```diff
@@ -1702,6 +1704,12 @@ EXPORT obs_scene_t *obs_group_from_source(const obs_source_t *source);
 EXPORT void obs_sceneitem_defer_group_resize_begin(obs_sceneitem_t *item);
 EXPORT void obs_sceneitem_defer_group_resize_end(obs_sceneitem_t *item);
 
+/** Gets the dsk from its source, or NULL if not a group */
```

Comment:
The word group at the end should be dsk here

Reply:
Fixed